Essentials/src/plugin.yml

```diff
@@ -571,3 +571,8 @@ permissions:
   essentials.near.exclude:
     default: false
     description: If the player should be excluded from near lookups.
+  essentials.togglejail:
+    default: false
```

Comment:
This line is unnecessary -- the correct value would be `OP`, which is already the default.

Reply:
Why remove `essentials.togglejail` check? If we wanted to remove this, the result would be false because it would only have ".jailName" permission.

I don't see why this would be unnecessary? The above permissions are also set to default and are also false. Players should not be denied the use of togglejail.

Reply:
You seem to not understand how superperms works.

What you're doing here is declaring a Permission to register, and setting attributes for it here. The other permissions declared are set to false because they need to be excluded from wildcards (`*`, `essentials.*`, etc) -- this one does not, so it can have its default value be OP

By giving `essentials.togglejail` children, when an admin grants `essentials.togglejail` in their permissions plugin, the children are set to the specified values as well.

Reply:
I'd also encourage you to take a look at https://bukkit.gamepedia.com/Plugin_YAML and https://www.spigotmc.org/wiki/plugin-yml/#permissions to learn more about this section of the plugin.yml

also: having `essentials.togglejail.offline` as child may introduce some quirks for new installations, but is unfortunately required to maintain compatibility :(